lib/ddtrace/workers.rb

```diff
@@ -87,6 +92,20 @@ def stop
         @run = false
       end
 
+      def shutdown!
```

Comment:
We should write a couple of lines to document what this method is doing. I'll let you describe this (i.e. "this method closes all available queues and wait till the trace buffer and service are flushed").